gradle.properties

```
@@ -0,0 +1,7 @@
+description='Kafka UI'
+version=0.1.0-SNAPSHOT
```

```suggestion
version=0.0.1-SNAPSHOT
```
to match the one in maven's pom

---

0.1.0 is more correct version in this case

---

why? in our poms it's 0.0.1